shoov/modules/custom/shoov_ci_build/shoov_ci_build.views.inc

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @file
+ * integration with the views module.
+ */
+
+/**
+ * Implements hook_views_data().
+ */
+function shoov_ci_build_views_data() {
+  $data['message']['trigger'] = array(
+    'title' => t('Trigger CI Build message'),
+    'help' => t('Link to send the message to nodejs server.'),
+    'field' => array(
+      'handler' => 'shoov_ci_build_trigger_button',
+    ),
+  );
+  return $data;
+}
+
+/**
+ * Implements hook_views_default_views_alter().
+ *
+ * Add link field to the view of messages to send the item to the nodejs server.
```

Comment:
I would keep it. If we want to add this link (or similar for other messages) in other message views - it makes it easier.